### DISCUSSION TITLE
Why not raise appearing if close modal page

### DISCUSSION THREAD

**Discussion by JaehwanMango** [dotnet/maui#10951] [Q&A]
Look i have 'A' Page
I want th this flow
1. Subscription when appearing the 'A' page
2. Unsubscribe when diappearing the 'A' Page

Open Modal Page on 'A' Page
It will call the Disappering in 'A'Page ViewModel

Close the Modal Page
It will not call Appearing in 'A'Page ViewModel

```
// 'A'PageViewModel

Appearing()
{
   // It will not call this when 'PopModalAsync();'
   Subscription;
}

Disappearing()
{
   // It will call this when 'PushModalPage();'
   Unsubscription;
}

PushModalPage()
{
  PushModalAsync();
}

```

**Comment by marcmognol** [comment]
Hi,
I think it is related to issue #8513
